examples/inline_types/src/types.ts

```diff
@@ -10,17 +10,18 @@ import {
     nat,
     nat64,
     query,
+    Record,
```

Comment:
The parsing should break if the type is not wrapped in `Record`.

Reply:
I've updated the parser to handle/require this.